cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,74 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import SettinsPageObject from '../support/pages/settingPageObjects';
+import homePageObject from '../support/pages/home.pageObject';
 
-  });
+const homePage = new homePageObject();
+const signInPage = new SignInPageObject();
+const settingsPage = new SettinsPageObject();
 
-  beforeEach(() => {
+const userData = {
+  username: 'riot',
+  email: 'riot@qa.team',
+  password: '12345Qwert!',
+};
 
+describe('Settings page', () => {
+  let user;
+
+  beforeEach(() => {
+    cy.task('db:clear');
+    homePage.visit();
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    settingsPage.loginUser();
```

Comment:
The loginUser method is called but the user data is not passed. The user data should be passed as an argument to 'loginUser' method.